specification/attestation/Attestation/client.tsp

```diff
@@ -0,0 +1,90 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.Attestation;
+
+@@clientName(JsonWebKey.x5c, "x5C");
+
+@@clientName(OpenIDConfigurationResponse.response_types_supported,
+  "responseTypesSupported"
+);
+@@clientName(OpenIDConfigurationResponse.id_token_signing_alg_values_supported,
+  "idTokenSigningAlgValuesSupported"
+);
```

Comment:
Why isnt this reversed? The camel case names are more idiomatic to tsp. If the wire names have the underscores you can use the `encodedName` decorator